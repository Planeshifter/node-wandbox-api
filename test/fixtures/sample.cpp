#include <iostream>

int main() {
	int x = 0;
	std::cout << "hoge" << std::endl;
}
