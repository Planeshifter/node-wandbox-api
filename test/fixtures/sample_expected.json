{
  "compiler_error": "prog.cc: In function 'int main()':\n<command-line>:0:3: warning: unused variable 'hogefuga' [-Wunused-variable]\nprog.cc:4:6: note: in expansion of macro 'x'\n  int x = 0;\n      ^\n",
  "compiler_message": "prog.cc: In function 'int main()':\n<command-line>:0:3: warning: unused variable 'hogefuga' [-Wunused-variable]\nprog.cc:4:6: note: in expansion of macro 'x'\n  int x = 0;\n      ^\n",
  "program_message": "hoge\n",
  "program_output": "hoge\n",
  "status": "0"
}
